web/app/js/components/DeploymentsList.jsx

```diff
@@ -79,40 +61,14 @@ export default class DeploymentsList extends React.Component {
         let meshDeploys = processRollupMetrics(rollup.metrics, "targetDeploy");
         let combinedMetrics = this.addDeploysWithNoMetrics(poByDeploy, meshDeploys);
 
-        this.loadTimeseriesFromServer(meshDeploys, combinedMetrics);
-      })
-      .catch(this.handleApiError);
-  }
-
-  loadTimeseriesFromServer(meshDeployMetrics, combinedMetrics) {
-    // fetch only the timeseries for the 3 deployments we display at the top of the page
-    let limitSparklineData = _.size(meshDeployMetrics) > maxTsToFetch;
-
-    let resourceInfo = this.api.urlsForResource["deployment"];
-    let mostActiveDeployments = this.getMostActiveDeployments(meshDeployMetrics);
-
-    let tsPromises = _.map(mostActiveDeployments, dep => {
-      let tsPathForDeploy = resourceInfo.url(dep.name).ts;
-      return this.api.fetchMetrics(tsPathForDeploy);
-    });
-
-    Promise.all(tsPromises)
-      .then(tsMetrics => {
-        let mostActiveTs = _.reduce(tsMetrics, (mem, ea) => {
-          mem = mem.concat(ea.metrics);
-          return mem;
-        }, []);
-        let tsByDeploy = processTimeseriesMetrics(mostActiveTs, resourceInfo.groupBy);
         this.setState({
-          timeseriesByDeploy: tsByDeploy,
-          lastUpdated: Date.now(),
           metrics: combinedMetrics,
-          limitSparklineData: limitSparklineData,
           loaded: true,
           pendingRequests: false,
           error: ''
         });
-      }).catch(this.handleApiError);
+      })
+      .catch(this.handleApiError);
```

Comment:
Did this `.catch()` move down one line on purpose?

Reply:
yeah, I kind of like this style.
we could consider adding `newline-per-chained-call` to enforce this.